pkg/json/decode.go

```diff
@@ -0,0 +1,97 @@
+package json
+
+import (
+	"strings"
+
+	"encoding/json"
+)
+
+func (u *Uuid) UnmarshalJSON(b []byte) error {
+
+	var s string
+	//todo:find better way to convert [,] to byte
```

Comment:
should be:

```
// TODO: find a better way to ...
```

notice TODO is in upper case (TODO, FIXME and BUG are searched throughput the code so it must be upper case.

notice the space after `//`

notice the space after `// TODO:'